pkg/rtc/transport.go

```diff
@@ -357,6 +358,7 @@ func newPeerConnection(params TransportParams, onBandwidthEstimator func(estimat
 		// As Firefox does not support migration, ICE Lite can be disabled.
 		se.SetLite(false)
 	}
+	se.SetDisableTransceiverReuseInRecvonly(params.DisableRecvonlyTransceiverReuse)
```

Comment:
This is the new flag to disable transceiver re-use when in `recvonly`. Details in pion PR - https://github.com/pion/webrtc/pull/3200. Will look at maintaining state in transceiver like Nils suggested later. In that case, this flag may not be required.